docs/pywbemlistener/generaloptions.rst

```diff
@@ -0,0 +1,78 @@
+.. _`pywbemlistener general options`:
+
+Pywbemlistener general options
+------------------------------
+
+General options are those that are specified directly after the ``pywbemlistener``
+command, e.g. ``--output-format simple`` in the following example:
+
+.. code-block:: text
+
+    pywbemlistener --output-format simple list
+
+Most of the general options can be specified with any of the sub-commands of the
+``pywbemlistener`` command, but there are some general options that are used
+only on some sub-commands, and are ignored on the other sub-commands.
+
+
+.. index:: single: pywbemlistener logging
+
+.. _`pywbemlistener logging`:
+
+Logging
+"""""""
+
+Logging is performed only by the ``pywbemlistener run`` command. The
```

Comment:
Is this important?  We are discussing the internals when what is important is that the logging logs indications received and state changes of the listener.

Reply:
`--logdir` is a global option, for consistency with pywbemcli. However, it is ignored on all commands but this one. That deserves some explanation.

Reply:
What we are trying to say is that only the run command actually adds entries to the log.  Or more general, the log is used only to provide information on the pywbemlistener operation based on python standard loging filters (ex: INFO, DEBUG, WARNING, ERROR, etc.)

Reply:
Yes, only the run command adds log entries.
No, the standard Python severities are not used, and not even the Python logging mechanism is used.